acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -0,0 +1,163 @@
+## These test the new trapperkeeper-authorization auth.conf default rules
+## as they're specified in FOSS puppet server.
+## The tests are written to assert the curl request was "not forbidden" rather
+## than expecting something meaningful from the endpoint. We're just trying to
+## test that authorization is allowing & rejecting requests as expected.
+##
+## The testing pattern is to call one of the two curl functions with a path,
+## and then one of the two assertion functions to validate allowed/denied.
+## The assertion functions look for regexes and status codes in the stdout
+## of the previous curl invocation.
+
+test_name 'Default auth.conf rules'
+
+step 'Turn on new auth support' do
+  modify_tk_config(
+    master,
+    File.join(master['puppetserver-confdir'], 'puppetserver.conf'),
```

Comment:
Oh, yeah it gets set to the proper path somewhere. FWIW this is used by `with_puppet_running_on`: https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/helpers/puppet_helpers.rb#L158